src/components/Widgets/Widget.tsx

```diff
@@ -81,9 +100,16 @@ export function Widget({ starterVariant }: WidgetProps) {
           setWalletSelectMenuState(true);
         },
       },
+      fromChain: fromChain,
+      fromToken: fromToken,
+      toChain: toChain,
+      toToken: toToken,
+      fromAmount: fromAmount,
```

Comment:
```suggestion
      fromChain,
      fromToken,
      toChain,
      toToken,
      fromAmount,
```

would do the same

Reply:
So true! Now, since we are re-using that interface - it´s deprecated and cleaner anyways. But good point!